internal/app/utils.go

```diff
@@ -109,7 +109,7 @@ func fromYAML(file string, s *state) (bool, string) {
 	if err = yaml.UnmarshalStrict([]byte(yamlFile), s); err != nil {
 		return false, err.Error()
 	}
-	addDefaultHelmRepos(s)
+	//addDefaultHelmRepos(s)
```

Comment:
nit: if the function was removed there's no point in keeping this commented out here.